mbuild/tests/test_packing.py

```diff
@@ -12,6 +12,14 @@ def test_fill_box(self, h2o):
         assert filled.n_particles == 50 * 3
         assert filled.n_bonds == 50 * 2
 
+    def test_fill_box_density_box(self, h2o):
+        filled = mb.fill_box(h2o, n_compounds=1000, density=1000)
+        assert [3.1 < period < 3.11 for period in filled.periodicity]
+
+    def test_fill_box_density_n_compounds(self, h2o):
+        filled = mb.fill_box(h2o, box=mb.Box([3.1, 3.1, 3.1]), density=1000)
+        assert 2900 < filled.n_particles < 3100
```

Comment:
I just wanted to leave a buffer in case something minor works on different computers, but this is unnecessarily large. I'll see if Travis passes the exact tests or I'll loosen them to a small range.